3_summarize.yml

```diff
@@ -120,3 +124,32 @@ targets:
       '3_summarize/src/annual_thermal_metrics.R')
     depends:
       - temp_ranges
+
+##-- GLM3 + GCM Projections Data Release --##
+
+  # Do all GCMs at once and combine into a single file
+  3_summarize/out/annual_metrics_glm3pb0_gcms.csv:
+    command: do_annual_metrics_multi_lake(
+      final_target = target_name,
+      site_file_yml = "1_fetch/out/pb0_temp_projections.yml",
+      ice_file_yml = I(NULL),
```

Comment:
Assume this is because the ice data is now included in the temperature feathers.

Reply:
Correct